config/routes.rb

```diff
@@ -0,0 +1,7 @@
+Rails.application.routes.draw do
+  scope(:path => '/api/v1') do
+    get 'welcome', to: 'root#welcome'
+  end
+
+  root 'root#index'
+end
```

Comment:
| rails s | ./bin/dev |
| --- | --- |
| ![image](https://github.com/Talha-Hanif5666/Rails-React-App/assets/93039359/415d51df-99a2-4e17-ae14-c911b4c41a6f) | ![image](https://github.com/Talha-Hanif5666/Rails-React-App/assets/93039359/6b7f1f6a-148f-4baa-9f83-30b8a1cc221b) |


- [ ] It would be nice if you see this and try to fix this error which I am facing on multiple computers 😨 Kindly fix this issue to make your app work as expected 👍

Reply:
Solved both issues. Please see images attached!
![railss](https://github.com/Talha-Hanif5666/Rails-React-App/assets/114238556/5e34c542-a5f4-4673-842f-5c61a0bc29db)

![bindev](https://github.com/Talha-Hanif5666/Rails-React-App/assets/114238556/ef25dfbb-d5af-4813-b5da-a6ec6d97679b)